PyInstaller/hooks/hook-distorm3.py

```diff
@@ -0,0 +1,16 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2005-2015, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+#By Starwarsfan2099
```

Comment:
@Starwarsfan2099 please add your name to `./doc/CREDITS.rst`, to the `PyInstaller 3.1` section.